server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -169,6 +169,8 @@ private ConfigProperties() {
     public static final String ENTITLER_JOB_THROTTLE =
         "pinsetter." + EntitlerJob.class.getName() + ".throttle";
 
+    public static final String BATCH_BIND_MAX_SIZE = "candlepin.batch.bind.max.size";
```

Comment:
Not a big deal, but I think I would name this `candlepin.bind.batch_size`.  Or at least drop the "max" out since the batch is always going to be the max size except at the tail end of a collection.